### ISSUE TITLE
doc: the link of document is broken

### ISSUE THREAD

**Issue by Aisuko** [burn-rs/burn#879]
**Describe the bug**

All the link of the documents are cannot be accessed. For example: https://burn.dev

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by louisfd** [comment]
@Aisuko I can't reproduce the problem, to me it looks fine. Can you give more details?

**Comment by Aisuko** [comment]
<img width="533" alt="Screenshot 2023-10-21 at 12 06 13 pm" src="https://github.com/burn-rs/burn/assets/8053949/0a5ac12a-7520-40a3-8387-6d6859365dba">

**Comment by nathanielsimard** [comment]
@Aisuko Is this resolved? I don't have any problems accessing the website from my end.

**Comment by Aisuko** [comment]
Hi @nathanielsimard I think it is because the reason below:

I'm trying to execute `curl burn.dev`

```
<script type="text/javascript">
var cat = "parked";
if (cat === "contract-cheating") {
    document.getElementById("header").innerHTML = "Violation of RMIT Policy";
    document.getElementById("message").innerHTML = "The website that you have attempted to access has been blocked by RMIT to ensure compliance with our <a href=https://policies.rmit.edu.au/document/view.php?id=168/>Academic Integrity policy</a>, <a href=https://policies.rmit.edu.au/document/view.php?id=35/>Student Conduct Policy</a> and the <a href=https://policies.rmit.edu.au/document/view.php?id=75>Information Technology and Security Policy</a>.";
    document.getElementById("message2").innerHTML = "This action aligns with the intentions of the Australian Government which has made it a criminal offence to provide or advertise academic cheating services in higher education, and which can lead to two years’ imprisonment or fines of up to $100,000.";
	document.getElementById("message3").innerHTML = "As such, RMIT University is committed to ensuring that our students are aware of these concerns, and is taking an educative approach to prevent, detect, and respond to academic cheating services.";
	document.getElementById("message4").innerHTML = "If you need further information, please contact <a href=mailto:student.conduct@rmit.edu.au>student.conduct@rmit.edu.au";
} else {
  document.getElementById("header").innerHTML = "Violation of the Information Technology and Security Policy";
  document.getElementById("message").innerHTML = "Access to the web page you are trying to visit has been logged and blocked in accordance with the RMIT Acceptable Use Standard - Information Technology.";
}
</script>
```

**Comment by louisfd** [comment]
Thanks, I'll close the issue since it seems to be related only to your university policy.
